src/yunohost/domain.py

```diff
@@ -113,20 +117,24 @@ def domain_add(auth, domain, dyndns=False):
             service_regen_conf(names=['nginx', 'metronome', 'dnsmasq'])
             app_ssowatconf(auth)
 
-    except:
+    except Exception, e:
+        from sys import exc_info;
+        t, v, tb = exc_info()
+
         # Force domain removal silently
         try:
             domain_remove(auth, domain, True)
         except:
             pass
-        raise
+        raise t, v, tb
```

Comment:
You don't need "t, v, tb" here, raise alone will reraise and keep the stack.

Reply:
I think I change that after a test in which the initial error line 120 was not reraise. I think if we put raise, it will raise the error from the remove operation no ?